src/style.css

```diff
@@ -33,6 +33,8 @@ h1 span {
 
 small {
   font-family: 'Press Start 2P', monospace;
+  font-weight: 600;
+  margin: 20px;
```

Comment:
Can we use utilities instead of this? So on the element add `font-semibold m-5` and remove these? That's the last thing, good to go then :)